matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/EventRelationsAggregationProcessor.kt

```diff
@@ -73,6 +74,7 @@ internal class EventRelationsAggregationProcessor @Inject constructor(
         private val sessionManager: SessionManager,
         private val liveLocationAggregationProcessor: LiveLocationAggregationProcessor,
         private val pollAggregationProcessor: PollAggregationProcessor,
+        private val encryptedReferenceAggregationProcessor: EncryptedReferenceAggregationProcessor,
         private val editValidator: EventEditValidator,
         private val clock: Clock,
 ) : EventInsertLiveProcessor {
```

Comment:
I wonder if we should not update the `shouldProcess()` methods and for UTD events just look if there is a `m.relates_to` field? 
Actually it could even be done at the `EventInsertLiveObserver` level? not sure why we would like to observe UTD events that are not relations?

Reply:
That's a good point, I agree, I will update the `EventInsertLiveObserver`.

Reply:
Done in [ee3cbd9](https://github.com/vector-im/element-android/pull/7851/commits/ee3cbd988481d0d071726831c720389fd60c176a)